addons/event/views/res_config_settings_views.xml

```diff
@@ -10,8 +10,8 @@
                 <xpath expr="//div[hasclass('settings')]" position="inside">
                     <div class="app_settings_block" data-string="Events" string="Events" data-key="event" groups="event.group_event_manager">
                         <h2>Events</h2>
-                        <div class="row mt16 o_settings_container">
-                            <div class="col-12 col-lg-6 o_setting_box" title="Add a navigation menu to your event web pages with schedule, tracks, a track proposal form, etc.">
+                        <div class="row mt16 o_settings_container" name="events_setting_container">
+                            <div class="col-12 col-lg-6 o_setting_box" id="manage_tracks" title="Add a navigation menu to your event web pages with schedule, tracks, a track proposal form, etc.">
```

Comment:
what is your rational for changes like this? why adding a `name` on one element and an `id` just below?

Reply:
@mart-e basically trying to keep it inline with what is done most on Odoo. I noticed that most of the `row` elements had a name and most of the `col` elements had an `id` for the existing records so I decided to keep it similar.

Reply:
@mart-e could you check my comment and give feedback? I'd rather not keep this PR open too long as it'll break quite a lot every time something in master changes. I'd rather not keep forward pushing all the time :D